routes/campaigners.rb

```diff
@@ -20,6 +29,54 @@ class Campaigners < Cuba
       res.redirect "/"
     end
 
+    on "petitions/new" do
+      render("petition/new",
+             title: "Actionite | Create New Petition",
+             form: partial('petition/form', petition: Petition.new))
+    end
+
+    on "petitions/save" do
+      on post do
+        params = req.POST
+        petition = (Petition.find_by_slug params['slug'] or Petition.new)
+        petition.name = params['name']
+        petition.title = params['title']
+        petition.slug = params['slug']
+        petition.body = params['body']
+        petition.mobile_body = params['mobile_body']
+        petition.facebook_title = params['facebook_title']
+        petition.language = params['language']
+        petition.alt_body = params['alt_body']
+        petition.image_url = params['image_url']
+        petition.image_text = params['image_text']
+        petition.suggested_tweets = params['suggested_tweets']
+        petition.goal = params['goal']
+        petition.auto_increment_goal = params['goal']
```

Comment:
Shouldn't it be?
`petition.auto_increment_goal = params['auto_increment_goal']`